lib/training/wiki_training_loader.rb

```diff
@@ -91,18 +91,25 @@ def training_content_and_translations(content:, base_page:, wikitext:)
     full_content
   end
 
-  # Gets a list of page titles linked from the base page
-  def wiki_source_pages
-    # To handle more than 500 pages linked from the source page,
-    # we'll need to update this to use 'continue'.
-    query_params = { prop: 'links', titles: @wiki_base_page, pllimit: 500 }
-    response = WikiApi.new(MetaWiki.new).query(query_params)
-    begin
-      response.data['pages'].values[0]['links'].map { |page| page['title'] }
-    rescue StandardError
-      raise InvalidWikiContentError, "could not get links from '#{@wiki_base_page}'"
+# Gets a list of page titles linked from the base page
+def wiki_source_pages
+  # To handle more than 500 pages linked from the source page,
+  # we'll need to update this to use 'continue'.
```

Comment:
If this PR implements the use of 'continue' then you should remove/update this comment